### PR TITLE
Follow instructions in DocumenterVitepress.jl readme (Take 2)

### PR DESCRIPTION
**Pull request by LilithHafner**
This version, specifically: https://github.com/LuxDL/DocumenterVitepress.jl/tree/80e5883f8cf20fa1926895c06c50898d3c9f8a24

### CODE REVIEW

**Comment by LilithHafner** [comment]
Page deploys but formatting is broken.

**Comment by asinghvi17** [comment]
Do you have Jekyll enabled on your repo by any chance?  This is really strange because I just added vitepress docs to [GeometryOps.jl](https://github.com/asinghvi17/GeometryOps.jl) and it is working quite well...

**Comment by LilithHafner** [comment]
I don't think so? Not unless it's enabled by default.

Screenshot from 2024-03-02 08-18-43

**Comment by LilithHafner** [comment]
The
```
 Warning: DocumenterVitepress: no assets directory found.
```
warning looks concerning. Tracing its origin

```julia
@warn "DocumenterVitepress: no assets directory found."
```
https://github.com/LuxDL/DocumenterVitepress.jl/blob/415f02f658024f672b7d80a41858c5cdd77ceb6a/src/writer.jl#L224

is run because the ASSETS directory is not found

```julia
 assets = ASSETS
 if isdir(assets)
```
https://github.com/LuxDL/DocumenterVitepress.jl/blob/415f02f658024f672b7d80a41858c5cdd77ceb6a/src/writer.jl#L213-L214

And the ASSETS directory is defined relative to the source code of DocumenterVitepress iteslf

```julia
const ASSETS = normpath(joinpath(@__DIR__, "..", "assets"))
```
https://github.com/LuxDL/DocumenterVitepress.jl/blob/415f02f658024f672b7d80a41858c5cdd77ceb6a/src/DocumenterVitepress.jl#L14

e.g. "/home/x/.julia/packages/DocumenterVitepress/OnzhP/assets" locally and https://github.com/LuxDL/DocumenterVitepress.jl/tree/415f02f658024f672b7d80a41858c5cdd77ceb6a/assets on github

It seems plausible that these missing assets could be the culprit.

**Comment by asinghvi17** [comment]
I just pushed some commits to my DocumenterVitepress branch which should solve this issue!  Turns out there was an extra `/` in the path I was providing Vitepress which complicated the whole affair.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -50,6 +50,8 @@ jobs:
         with:
           version: '1'
       - uses: julia-actions/julia-buildpkg@v1
+      - name: 'Install experimental packages'
+        run: julia --project=docs -e 'using Pkg; pkg"add DocumenterVitepress#asinghvi17-patch-3"'
```
Comment (asinghvi17) [inline .github/workflows/CI.yml:54]:
```suggestion
        run: julia --project=docs -e 'using Pkg; pkg"add DocumenterVitepress#master"'

**Comment by LilithHafner** [comment]
It works!!!

**Comment by asinghvi17** [comment]
Finally 😅 ... looks like the issue was just bad support for custom domains which should now be fixed!

**Pull request merged**